py-bin/README.md

```diff
@@ -0,0 +1,12 @@
+# Py-Bin
+
+This directory contains an npm and a python package used to pack the `contracts.json` file containing among other things
+the abi and bytecode of all compiled contracts from this repository.
+
+The `tlbin` python package can be used to easily load the `contracts.json` with the following:
+
+```python
+from tlbin.contracts import load_packaged_contracts
```

Comment:
I think it would be nicer to be able to do `from tlbin import` which is easy to do via `__init__` for example.